### PR TITLE
Code clean up

### PR DESCRIPTION
**Pull request by brianwchou**
## Description
The game file can be reduced in size. Lets see if we can make it around a 100 lines

## Todo
- [x] code reformat

### CODE REVIEW

File: web/src/Game.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React, { useState, useEffect, useContext } from 'react';
 import { EthersContext } from './EthersContext.js';
+import UsedChars from './UsedChars';
+import WinState from './WinState';
+import GuessCounter from './GuessCounter';
```
Comment (ejwessel) [inline web/src/Game.js]: @brianwchou did you forget to include these files?

File: web/src/Game.js [hunk 2]
```diff
@@ -57,16 +59,10 @@ function Game() {
...
-    hexChars.forEach((hex) => {
-      if(hex !== "0x00") {
-        result.push(window.web3.toAscii(hex));
-      } else {
-        result.push("_");
-      }
-    })
+    let result = hexChars.map( hex => {
+        return hex !== "0x00" ? window.web3.toAscii(hex) : "_";
+    }).join(" ");
```
Comment (ejwessel) [inline web/src/Game.js]: nice

File: web/src/Game.js [hunk 3]
```diff
@@ -75,21 +71,15 @@ function Game() {
...
```
Comment (ejwessel) [inline web/src/Game.js]: nice

File: web/src/Game.js (hunk 1 above)
Comment (brianwchou) [reply web/src/Game.js]: No i took out those components and made it their own file.

**Pull request merged**